dist/screeps.d.ts

```diff
@@ -1216,7 +1216,7 @@ interface PathFinder {
      *
      * @param isEnabled Whether to activate the new pathfinder or deactivate.
      */
-    use(isEnabled: boolean): any;
+    use(isEnabled: boolean): undefined;
```

Comment:
I can get behind that; let's bring it in across all applicable functions in another PR.